NEWS.md

```diff
@@ -1,6 +1,6 @@
 # NEWS
 
-## Polars R Package (development version)
+## Polars R Package 0.16.3
```

Comment:
Can you mention the new URL in the news and that old links are redirected?

Reply:
Thanks for your quick review.
How about b768b3d37fbad070a41d337937b1da86ef300fab?

Reply:
By the way, I wanted to release this earlier because the automatic link by downlit was broken as shown below. (The redirect is to the top page, so downlit can no longer find pkgdown.yml)

https://github.com/rpolars/r-polarssql/commit/c4c6478a1d64cfd4f3ff003815476859a84bea34#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R119

Reply:
Thanks, can you just mention the old and new root URLs in the news?

Reply:
How about c816cd6194be24e5124256b88498b5d4a72d2862?

I did not do this because I thought it might cause an error if the URL check by R CMD check detected a redirect, but perhaps it does not have to be an actual link here.

Reply:
Ah i didn't think about that, let's remove it if R CMD check errors. Thanks!

Reply:
It passed the tests.